src/NameGenerator.php

```diff
@@ -26,9 +26,13 @@ public function createProjectServiceAccountId(string $projectId): string
         return sprintf('%s-%s', $this->stackPrefix, $projectId);
     }
 
-    public function createObjectNameForBucketInProject(string $bucketId): string
+    public function createObjectNameForBucketInProject(string $bucketId, ?string $branchId): string
     {
-        return str_replace(['.', '-'], '_', $bucketId);
+        $bucketId = str_replace(['.', '-'], '_', $bucketId);
+        if ($branchId !== null) {
+            return sprintf('%s_%s', $branchId, $bucketId);
+        }
+        return $bucketId;
```

Comment:
udělal sem to optional, pro default branch sa tam branchId nepřidává, takže ju nebudu posílat z connection